espn_api/basketball/activity.py

```diff
@@ -1,28 +1,43 @@
-from .constant import ACTIVITY_MAP
+from .constant import ACTIVITY_MAP, POSITION_MAP
 
 class Activity(object):
-    def __init__(self, data, player_map, get_team_data):
+    def __init__(self, data, player_map, get_team_data, include_moved=False):
         self.actions = [] # List of tuples (Team, action, player)
         self.date = data['date']
         for msg in data['messages']:
             team = ''
             action = 'UNKNOWN'
             player = ''
+            position = ''
             msg_id = msg['messageTypeId']
             if msg_id == 244:
                 team = get_team_data(msg['from'])
             elif msg_id == 239:
                 team = get_team_data(msg['for'])
+            elif msg_id == 188:
+                if include_moved:

```

Comment:
these checks can be moved to the top elif `msg_id == 188` and `and` together

Reply:
Makes sense